src/components/StepIndicator/StepIndicator.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import StepIndicator, {StepIndicatorProps} from '.';
+
+const baseProps: StepIndicatorProps = {
+  number: 1,
+  text: <div>Hello world</div>,
+};
+
+describe('StepIndicator', () => {
+  it('renders proper default className', () => {
+    render(<StepIndicator {...baseProps} />);
+    const el = screen.getByTestId('StepIndicator');
+
+    expect(el.className).toBe('StepIndicator');
```

Comment:
can you add tests here for the children div's too?

Reply:
yep done so